src/components/Modal/Modal.js

```diff
@@ -0,0 +1,31 @@
+import { Modal as MuiModal } from '@material-ui/core';
+import React from 'react';
+import { StyledModalBtn, StyledModalLayout } from './elements';
+
+const Modal = ({
```

Comment:
This is now a semi-generic Modal component that could be used in a number of situations.